style.css

```diff
@@ -0,0 +1,234 @@
+.body {
+    background-color: #f8fafc;
+    margin: 0 auto;
+}
+
+.header {
+    background-color: white;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 100px;
+    font-family: 'Roboto', sans-serif;
+    letter-spacing: 2px;
+    padding: 0px 0px 0px 200px;
+    box-shadow: 0 5px 8px -2px lightgray;
+}
+.pavadinimas {
+    float: left;
+}
+span {
+    color: #0000ff;
+}
+.fa-brands {
+    color: blue;
+    padding: 0px 10px;
+}
+.icons {
+    padding: 0px 200px 0px 40px;
+}
+.fa-brands:hover {
+    color: lightskyblue;
+}
+input:focus {
+    border: 2px solid black;
+}
+.task-input {
+    font-family: 'Roboto', sans-serif;
+    background-color: white;
+    height: 200px;
+    width: 600px;
+    margin: 100px auto 50px;
+    box-shadow: 0 5px 8px 1px lightgray;
+    border-radius: 5px;
+}
+.form {
+    display: flex;
+    justify-content: center;
+}
+#task {
+    width: 300px;
+    height: 30px;
+    border-radius: 5px 0px 0px 5px;
+    border-style: solid none solid solid;
+    border-color: lightgray;
+    border-width: 1px;
+    background-color: #f8fafc;
+}
+::placeholder {
+    color: lightgray;
+    font-weight: 300;
+    font-size: larger;
+    padding-left: 5px;
+}
+.add {
```

Comment:
it's a bit difficult to tell what this is based on the class name. `.add-btn` or `add-button` would be more appropriate